app/javascript/readingList/readingList.jsx

```diff
@@ -88,14 +89,9 @@ export class ReadingList extends Component {
     event.preventDefault();
 
     const { statusView, items, totalCount } = this.state;
-    window.fetch(`/reading_list_items/${item.id}`, {
+    request(`/reading_list_items/${item.id}`, {
```

Comment:
This should be awaited and then continue based on a successful request, but that will come in another refactor PR.